mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
```

Comment:
Add mypy in separate PR.

Reply:
I want to merge mypy in this PR and then make a separate PR in which I will add typing's to all existing functions.